additional clarification around trailing parameters (#116)
ircdocs/modern-irc: index.md

      middle      =  nospcrlfcl *( ":" / nospcrlfcl )
      trailing    =  *( ":" / " " / nospcrlfcl )

> Parameters are a series of values separated by one or more ASCII SPACE characters `(' ', 0x20)`. However, this syntax is insufficient in two cases: a parameter that contains one or more spaces, and an empty parameter. In order to permit such parameters, the final parameter can be prepended with a `(':', 0x3A)` character, in which case that character is stripped and the rest of the message is treated as the final parameter, including any spaces it contains. Parameters that contain spaces, are empty, or begin with a `':'` character MUST be sent as a final parameter with a preceding `':'`; in other cases, the use of a preceding `':'` for the final parameter is OPTIONAL.

Reviewer: ```suggestion
Parameters are a series of values separated by one or more ASCII SPACE characters `(' ', 0x20)`. However, this syntax is insufficient in two cases: a parameter that contains one or more spaces, and an empty parameter. To permit such parameters, the final parameter can be prepended with a `(':', 0x3A)` character, in which case that character is stripped and the rest of the message is treated as the final parameter, including any spaces it contains. Parameters that contain spaces, are empty, or begin with a `':'` character MUST be sent with a preceding `':'`; in other cases the use of a preceding `':'` on the final parameter is OPTIONAL.
```

Minor copyedits to make the paragraph easier to read.

      :dan!d@localhost PRIVMSG #chan Hey!   ->  ["#chan", "Hey!"]

>       :dan!d@localhost PRIVMSG #chan ::-)   ->  ["#chan", ":-)"]
> 
> As these examples show, a trailing parameter (a final parameter with a preceding `':'`) has the same semantics as any other parameter, and MUST NOT be treated specially once the `':'` is stripped.

Reviewer: ```suggestion
      :dan!d@localhost PRIVMSG #chan ::-)   ->  ["#chan", ":-)"]

As these examples show, a trailing parameter (a final parameter with a preceding `':'`) has the same semantics as any other parameter, and MUST NOT be treated specially or stored separately once the `':'` is stripped.
```

Minor copyediting, and clarification in terms of 'stored separately' on the final bit of the explanation.
